project/project.go

```diff
@@ -19,6 +20,8 @@ var (
 	file ini.File
 
 	pp = ""
+
+	copyToRoot = flag.Bool("r", false, "set root of the project as build out. should not be used with watch")
```

Comment:
It'll start going in an infinite loop if you are compiling and it detects changes and fires a recompilation